docker.Makefile

```diff
@@ -18,7 +18,7 @@ CUDA_CHANNEL              = nvidia
 # The conda channel to use to install pytorch / torchvision
 INSTALL_CHANNEL          ?= pytorch
 
-PYTHON_VERSION           ?= 3.8
+PYTHON_VERSION           ?= 3.9
```

Comment:
Can we change it to 3.10 please (@desertfire asked me about Python-3.10 in our docker images to run some of the models)
```suggestion
PYTHON_VERSION           ?= 3.10
```